### PR TITLE
notask readme Updated naming conventions for branches and commits

### PR DESCRIPTION
**Pull request by artem-solovev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
 
 Join us to come with us
+___
+
+#### !!! Git branches and commits patterns
+
+###### branch name format:
+**[name]/[type]/[GITHUB-TASK] [short-description]**<br>
+e.g.<br> 
+artem/feature/#33-some-module-feature<br>
```
Comment (HolyPawer) [inline README.md]: CHLEN NE DO KOLEN
PS POkA CHTO

Comment (HolyPawer) [reply README.md]: 123

File: README.md [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
+igor/notask/#999-some-screen-issue<br>
+
+###### commit message format:
+**[JIRA-TASK] [Module] - [work description]**<br>
+e.g.<br>
+#333 App - added layout props to View<br>
+#555 Catalog - product card for search result list
+
+###### merge request title format (used in app change log):
+**[type]/[JIRA-TASK] [app change description]**<br>
```
Comment (HolyPawer) [inline README.md]: NORMAL'NO BLYAT

Comment (artem-solovev) [reply README.md]: Thanks idiot

**Pull request merged**